astronomer/providers/apache/hive/hooks/hive.py

```diff
@@ -17,17 +18,28 @@ class HiveCliHookAsync(BaseHook):
 
     def __init__(self, metastore_conn_id: str) -> None:
         """Get the connection parameters separated from connection string"""
-        self.metastore_conn_id = self.get_connection(conn_id=metastore_conn_id)
```

Comment:
There are 3 ways(using thrift connection , metastore and hive cli) to  make connection to Hive in Airflow. 

- What are the sync versions of these sensors using?  
- Have these sensors been tested using same mechanism as sync sensors

Reply:
Both `HivePartitionSensor` and `NamedHivePartitionSensor` uses Hive Metastore thrift connection type and same as been used in `HivePartitionSensorAsync` and  `NamedHivePartitionSensorAsync` and has been tested for the connection.